src/components/tab/tabs.stories.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { Tabs, Tab } from ".";
+
+export default { title: "Tabs" };
```

Comment:
Please add the component name so the storybook docs addon can work effectively, see #41

```js
export default {
    title: 'Tabs',
    component: 'Tabs'
};
```